config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "locals": {
+    "url": "http://engineering.underdog.io",
+    "name": "Underdog.io Engineering blog",
+    "owner": "Underdog.io Engineering",
+    "description": "Articles and insights from the Underdog.io Engineering team"
+  },
+  "plugins": [
+    "./plugins/paginator.coffee"
+  ],
+  "require": {
+    "moment": "moment",
+    "_": "underscore",
+    "typogr": "typogr"
+  },
+  "jade": {
+    "pretty": true
+  },
+  "markdown": {
+    "smartLists": true,
+    "smartypants": true
```

Comment:
What does this enable that we never had before? It looks like code blocks are getting parsed and highlighted.